spec/models/wikipedia_artist_fetcher_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'vcr'
+
+VCR.configure do |c|
```

Comment:
@solipet how do you feel about moving this to `spec_helper`? I think it's nicer to have stuff like this available in any test.